frontend/src/components/activePage/ActiveView/ActiveView.tsx

```diff
@@ -60,6 +69,54 @@ const ActiveView: FC<IActiveViewProps> = ({ ...props }) => {
                 }}
                 enterButton
               />
+              <Button
+                className="hidden xl:block"
+                type="primary"
+                shape="round"
+                size="middle"
+                icon={<FullscreenOutlined />}
+                onClick={() => {
+                  setExpandAll(true);
+                }}
```

Comment:
```suggestion
                onClick={() => setExpandAll(true)}
```
Also on the other cases below